README.md

```diff
@@ -15,7 +15,7 @@ container agnostic and provide portability when multiple container deployments a
 Usage
 ---------------
 
-The first step is to simply [resolve](http://opensource.morganstanley.com/desktopJS/modules/_desktopjs_src_registry_.html#resolvecontainer) the current [container](http://opensource.morganstanley.com/desktopJS/classes/_desktopjs_src_container_.container.html). This enumerates
+The first step is to simply [resolve](http://morganstanley.github.io/desktopJS/modules/_desktopjs_src_registry_.html#resolvecontainer) the current [container](http://morganstanley.github.io/desktopJS/classes/_desktopjs_src_container_.container.html). This enumerates
```

Comment:
I get page not found error(File not found The site configured at this address does not contain the requested file.) for both of these urls:
http://morganstanley.github.io/desktopJS/classes/_desktopjs_src_container_.container.html
http://morganstanley.github.io/desktopJS/modules/_desktopjs_src_registry_.html#resolvecontainer

and for the current ones as well:
http://opensource.morganstanley.com/desktopJS/modules/_desktopjs_src_registry_.html#resolvecontainer
http://opensource.morganstanley.com/desktopJS/classes/_desktopjs_src_container_.container.html

is that expected?

Reply:
@lilla28  Great catch unfortunately of a problem that was there before even on other url.  Let me update in this PR.

Reply:
It works now, thanks!